src/modules/identity/RadixLedgerIdentity.ts

```diff
@@ -0,0 +1,249 @@
+/*
+ * (C) Copyright 2020 Radix DLT Ltd
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the “Software”),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+
+import { RadixAccount, RadixIdentity } from '../..'
+import { RadixAddress, RadixAtom } from '../atommodel'
+import TransportNodeHid from '@ledgerhq/hw-transport-node-hid'
+import {
+    Instruction,
+    Handler,
+    SignPayloadType,
+    ReturnCode,
+} from './ledgerAppTypes'
+
+const CLA = 0xaa
+
+// HD Derivation Path: /0'/0/0
+const BIP44_PATH = '80000000' + '00000000' + '00000000'
+
+const CHUNK_SIZE = 250
+
+export default class RadixLedgerIdentity implements RadixIdentity {
```

Comment:
I think the naming is confusing, seems like it might refer to the RadixLedger component

Reply:
Yeah I was thinking about this. Might use HardwareWallet instead?

Reply:
We might implement other types of hardware wallets in the future, maybe HardwareWallerLedgerNano (if this would work for all types of ledger, otherwise LedgeNanoS even)

Reply:
How about HWLedger? as in Hardware Wallet Ledger